File: scripts/timed/on_tick.js
//var t = handler.get_timer("test", 10);
state["timer_test"] = core_timer.is_running();

if(env.datetime.year == 2011) {
    state["wooooo"] = "12345";
}

if(env.datetime.second > 40) {
    handler.add_action("test_provider", "test_action", {"p1": "test"});
}

function hello() {
    for(var i=0; i<10; i++) {
        state["hello"+i] = i
    }
}

hello();

File: scripts/timed/on_tick.py

bbs_a1 = state.get("bbs_a1", False)

#send_message("HELLo", key="input.test")

dt = env.get("datetime")
new_minute, new_second = dt.get("new_minute"), dt.get("new_second")

if new_minute: # and dt.get("hour_minute") == "2028":
    #send_message({"action": "byebyestandby", "params": {"device": "a1", "state": not bbs_a1}})
    state["bbs_a1"] = not bbs_a1
